dspace/config/modules/orcid.cfg

```diff
@@ -0,0 +1,24 @@
+#------------------------------------------------------------------#
+#--------------------ORCID GENERIC CONFIGURATIONS------------------#
+#------------------------------------------------------------------#
+
+#Allowed values are disabled, only_admin, only_owner or admin_and_owner
+orcid.disconnection.allowed-users = admin_and_owner
```

Comment:
Please describe the values in more detail here.  Does `disabled` and `only_owner` mean that Administrators will not be able to delete the EPerson?  Or do they just block the disconnect of ORCID in some way?